public/js/question.js

```diff
@@ -44,7 +63,7 @@ var sendAnswerRequest = function(ans) {
                     if(getRating() > 0) {
                         submitQuestionRating(getRating(), questionId);
                     }
-                    window.location.href = '/';
+                    location.reload();
```

Comment:
so now we're staying in the question right? so that they can discuss?

Reply:
yup!